core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -45,6 +45,34 @@ oppia.factory('explorationData', [
     // Put exploration variables here.
     var explorationData = {
       explorationId: explorationId,
+      autosaveChangeList: function(changeList, successCallback, errorCallback) {
+        var draftAutosaveUrl = '/createhandler/autosave_draft/' + explorationId;
```

Comment:
I don't think you need this variable; just write directly into the if statement. Likewise below.